integrations/wootric/lib/index.js

```diff
@@ -34,6 +33,7 @@ Wootric.prototype.initialize = function() {
   this.lastPageTracked = null;
   window.wootricSettings = window.wootricSettings || {};
   window.wootricSettings.account_token = this.options.accountToken;
+  window.wootricSettings.version = 'segment-beacon-2.3.0';
```

Comment:
Why is the version number 2.3.0? Didn't we just added this into the main branch? IIRC it's set to 1.0.0. Can we have a uniform version? the only difference should be `segment-beacon`

Also, is this name approved by Prabhat? why not `segment-wootric-sdk`? since the `beacon` keyword is not the official name of this

Reply:
I'll ask Prabhat

Reply:
Ok, Prabhat suggested ` wootric-segment-js-x.y.z`